opencti-platform/opencti-front/src/private/components/entities/sectors/SectorKnowledge.js

```diff
@@ -64,6 +64,28 @@ class SectorKnowledgeComponent extends Component {
               />
             )}
           />
+          <Route
+            exact
+            path="/dashboard/entities/sectors/:sectorId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={sector.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
```

Comment:
According to the stix.ts file, it's missing the type Attack-Pattern.

Reply:
Fixed